src/utils/dompurify/index.js

```diff
@@ -0,0 +1,5 @@
+import dompurify from 'dompurify'
+
+export function safe(html) {
```

Comment:
```suggestion
export function sanitize(html) {
```

Desculpe pedir mais uma alteração aqui, mas usamos `__safe()` em vários projetos de front e eles escapam html. Não é o que acontece aqui, estamos só sanitizando. Para não confundir com coisas existentes, vamos chamar o método de `sanitize()` aqui para deixar claro que é  algo diferente.

Reply:
Não sabia da existência do `__safe()`, renomeei para `sanitize()`, obrigado por fazer esse apontamento.